source/deploy/legacy-cloud-changelog.md

```diff
@@ -2028,7 +2028,7 @@ From Mattermost v9.2, see the [Mattermost Changelog](https://docs.mattermost.com
  - Added a global header for product navigation for Channels, Playbooks, and Boards. This is disabled on the mobile web view and mobile apps.
 
 #### Packaging Changes
- - Introducing [updated packaging](https://mattermost.com/pricing) with new Starter, Professional and Enterprise plans to better serve our customers.
+ - Introducing [updated packaging](https://mattermost.com/pricing) with new Starter, Enterprise and Professional plans to better serve our customers.
```

Comment:
We don't have a Starter plan any longer.

Reply:
Ah, I see this is in the change log. I think we are good without any changes here.